ydb/core/tx/schemeshard/schemeshard_impl.cpp

```diff
@@ -6162,6 +6200,8 @@ void TSchemeShard::Handle(TEvSchemeShard::TEvModifySchemeTransactionResult::TPtr
         return Execute(CreateTxReply(ev), ctx);
     }
 
+    HandleBackgroundCleaningTransactionResult(ev);
```

Comment:
Inconsistent handling -- any BackgroundCleaningTransaction will result in warning `no able to determine destination ...`
`BackgroundCleaningTxs` should be checked here the same way as other `TxIdTo...` mappings -- explicitly and with immediate return.